src/issues/issue-2/dj-ban.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import Img from "gatsby-image"
+import { useStaticQuery, graphql } from "gatsby"
+
+const DJBan = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      file(relativePath: { eq: "issue2-images/dj-ban.png" }) {
+        childImageSharp {
+          fluid {
+            ...GatsbyImageSharpFluid_withWebp_tracedSVG
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <Img style={{ zIndex: "1000" }} fluid={data.file.childImageSharp.fluid} />
```

Comment:
Needs to have high z index in order to cover the `New York` text from section above